docs/ibexa_engage/install_ibexa_engage.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Install and configure Ibexa Engage.
+edition: experience
+---
+
+# [[= product_name_engage =]]
+
+[[= product_name_engage =]] is a data collection tool. It enables you to engage your audiences by using the [Qualifio](https://qualifio.com/) tools.
+You can use interactive content to gather valuable data, for example, customer personal data or recent orders list, and create connections.
+
+For more information, see [Qualifio Developers documentation](https://developers.qualifio.com/docs/engage/).
+
+## Enable [[= product_name_engage =]] account
+
+To use [[= product_name_engage =]], you must make arrangements with [[= product_name =]] to define the initial configuration.
+[[= product_name_base =]] team creates and provides user account. An invitation link is sent during the setup process.
+
+For more information, see [Ibexa Engage in User documentation]([[= user_doc =]]/ibexa_engage/ibexa_engage/#request-access).
+
+## Install [[= product_name_engage =]]
+
+[[= product_name_engage =]] comes with v4.6.6 of [[= product_name_exp=]].
+If you have different version, run the following command to install the bundle:
```

Comment:
You can check for it's presence by running
```bash
composer show | grep "ibexa/engage"
```
(or using `composer info` and checking the list manually).

The package will be included in 4.6.6 and any above :)